telluric/util/raster_utils.py

```diff
@@ -65,22 +65,42 @@ def _get_telluric_tags(source_file):
         return return_tags
 
 
-def convert_to_cog(source_file, destination_file, resampling=Resampling.gauss):
+def _create_options_for_cog(create_option, source_profile, blocksize):
+    """
+    it useses the profile of the source raster, override anything using the creation_options
+    and gerantees we will have tiled raster and blocksize
+    """
+    if not(create_option):
+        create_option = {}
+
+    create_option["blocksize"] = blocksize
+    create_option["tiled"] = True
+    for key in ["nodata", "compress"]:
+        if key not in create_option:
+            create_option[key] = source_profile.get(key, None)
+    return create_option
+
+
+def convert_to_cog(source_file, destination_file, resampling=rasterio.enums.Resampling.gauss, blocksize=256,
```

Comment:
Is `create_option` a dictionary? If so, we should add a docstring and probably rename it to `create_options`

Reply:
check out the docstring of `convert_to_cog`